documentation/writing_plans.md

```diff
@@ -1094,6 +1094,25 @@ MODULE:
 /Users/bolt/.puppetlabs/bolt
 ```
 
+## Making plans private
+
+As a plan author, you may no want users to run your plan directly or know it exists. This is useful
```

Comment:
Ugh I know this misspelled but I don't have the heart to change it rn I will tomorrow.
```suggestion
As a plan author, you may not want users to run your plan directly or know it exists. This is useful
```